lib/pre-commit/utils/staged_files.rb

```diff
@@ -1,10 +1,19 @@
-require "pre-commit/configuration/top_level"
+# frozen-string-literal: true
+
+require 'pre-commit/configuration/top_level'
 
 module PreCommit
   module Utils
     module StagedFiles
       include PreCommit::Configuration::TopLevel
 
+      BINARIES = [".img", ".iso", ".dmg"].freeze
+      DOCUMENTS = [".md"].freeze
```

Comment:
not sure we should do that, I can easily imagine whitespace, structure or spell check on markdown